src/sentry/static/sentry/app/components/gridEditable/index.tsx

```diff
@@ -0,0 +1,12 @@
+import Grid from './grid';
```

Comment:
I think we've decided against using `index.jsx` to re-export components, see https://www.notion.so/sentry/321690a3061f43fe85f66df25435d883?v=daa97633c42242d4acc818b140afdd7f&p=5f9aef105d384e6da5fe873f309e3a98